### PR TITLE
Move to stable version

### PR DESCRIPTION
**Pull request by LTA-Thinking**
Moved from a preview version to a stable version. Added a new example, cleaned up some existing examples, and modified some descriptions text. 

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ x ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ x ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ x ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ x ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/healthcareapis/resource-manager/readme.md [hunk 1]
```diff
@@ -28,10 +28,19 @@ These are the global settings for HealthcareApis service.
 title: HealthcareApisManagementClient
 description: Azure Healthcare APIs Client
 openapi-type: arm
-tag: package-2018-08-preview
+tag: package-2019-09-16
 azure-arm: true
 ```
 
+### Tag: package-2018-08-preview
```
Comment (nschonni) [inline specification/healthcareapis/resource-manager/readme.md]:
```suggestion
### Tag: package-2019-09-16
```

File: specification/healthcareapis/resource-manager/readme.md [hunk 2]
```diff
@@ -128,6 +150,7 @@ require: $(this-folder)/../../../profiles/readme.md
 
 # all the input files across all versions
 input-file:
+   - $(this-folder)/Microsoft.HealthcareApis/stable/2019-09-16/healthcare-apis.json
```
Comment (michaeljqzq) [inline specification/healthcareapis/resource-manager/readme.md]: format error

**Comment by poadhika** [comment]
@nschonni @michaeljqzq  I saw the label WaitForArmFeedback. Is there any action that we need to take from our side? We have a GA deadline on 9-16. We need to complete this work before that timeframe. Any help would be mush appreciated.

**Comment by michaeljqzq** [comment]
@poadhika ARM team will review the PR, normally need no more than a week

**Comment by michaeljqzq** [comment]
@openapibot sdkautomation rebuild

**Comment by poadhika** [comment]
@michaeljqzq Thank you for your response. We have a GA deadline of September 16. How can we expedite this process?

**Review by anthony-c-martin — CHANGES_REQUESTED** [review]

**Review by michaeljqzq — APPROVED** [review]

**Comment by yungezz** [comment]
hi @anthony-c-martin could you pls change review state from "request change" to "approve" so we can go ahead to merge?

**Review by anthony-c-martin — APPROVED** [review]

**Comment by poadhika** [comment]
@michaeljqzq, @nschonni : Can you please merge the PR?

**Pull request merged**